Add metapackage migration guide (#2611)
Qiskit/documentation: docs/faq.mdx

These separate software packages were combined into what was called the Qiskit *metapackage*, which allowed users to install all of these components together through `pip install qiskit`. This metapackage architecture was removed with the release of the Qiskit SDK v1.0, and now each of the individual software distributions needs to be installed separately. 

> If you want to install the packages that used to be included in the Qiskit metapackage, visit the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) to find a range of packages to suit your needs. You can also read the [v1.0 migration guide](/migration-guides/metapackage-migration) for more information about this change.

Reviewer: ```suggestion
If you want to install the packages that were previously included in the Qiskit metapackage, visit the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) to find a range of packages to suit your needs. You can also read the [v1.0 migration guide](/migration-guides/metapackage-migration) for more information about this change.
```
